app/gui/view/graph-editor/src/component/visualization/container.rs

```diff
@@ -278,11 +282,15 @@ impl View {
     }
 
     fn init(self) -> Self {
-        self.init_background();
+        let styles = StyleWatch::new(&self.scene.style_sheet);
```

Comment:
The `FIXME` comment was lost. You can also take the chance and change it to StyleWatchFRP (but it would need some FRP nodes for refreshing).

Reply:
Restored the fixme comment, also used FRP for *part* of the stuff in this method. The main stopper is shadow for dom, I don't think we have FRP-based API for that at the moment.